application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java

```diff
@@ -9,11 +9,10 @@
  */
 enum UserStrings {
     NEW_QUESTION("""
-            Thank you for asking a question in an available channel.
-            When a helper who can answer this question reads it they will help you.
+            Thank you for asking in an available channel. A helper will be with you shortly!
             Please be patient.
 
-            __Please do not post your question in other channels__
+            __**Do not post your question in other channels.**__
```

Comment:
I think the **bold** is a bit too much and takes away attention from the core message.

```suggestion
            __Do not post your question in other channels.__
```